src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -172,14 +172,13 @@
          (get-in ws ["workspace" "bucketName"])])
       (style/create-section-header "Analysis Submissions")
       (style/create-paragraph
-        (let [count-all (count submissions)
-              grouped-by-status (group-by #(% "status") submissions)]
+        (let [count-all (reduce + 0 (vals submissions-count))]
```

Comment:
`(reduce + 0 some-list)` is effectively equivalent to `(apply + some-list)`.  It works just fine if the list is empty or nil.  `apply` should be a hair faster, but it doesn't matter.  (shrug emoji)